### DISCUSSION TITLE
How to display variable values right next to the code when stepping in a debugger?

### DISCUSSION THREAD

**Discussion by MortalWombat-repo** [neovim/neovim#24103] [Q&A] [answered]
Similar to this?

https://www.jetbrains.com/help/rider/Inspecting_Variables.html#inline-with-code
Thanks :)

**Comment by yutkat** [comment, marked as answer]
https://github.com/theHamsta/nvim-dap-virtual-text

**Reply by MortalWombat-repo** [reply]
Thank you i saw that will that behave okay with dap ui?

**Reply by nyngwang** [reply]
@Ico70 Good question. I encountered an issue before but it had been fixed: https://github.com/theHamsta/nvim-dap-virtual-text/issues/57. Other than this, the answer should be yes.
